text/0031-designated-operators.md

```diff
@@ -18,13 +18,46 @@ It is expected that many validators will want to rely on third parties to provis
+At bid creation, the prospective validator may choose to specify an operator, by passing in the operator's public key. If the bid wins the auction, the operator will be responsible for creating all messages, signing with own private key. However, the validator will collect all of the rewards and would be subject to slashing, as usual. Our assumptions is that an off-chain contractual relationship provides for the operator's compensation. Existing validators and bidders are able to switch operators at will, with the change taking effect in the first future era where the new operator is specified in a validator's winning bid. There is no confirmation for the relationship, but provisions have to be made to restrict designation to simplify reasoning about the behavior of the validator-operator pair. 
 
 ## Reference-level explanation
 
 [reference-level-explanation]: #reference-level-explanation
 
-N/a
+### User workflow changes
+
+Add an additional field, expecting a public key (or some type wrapping a public key), to the `add_bid` entry point of the auction contract. The supplied public key is expected to be part of a key pair belonging to a third party ("third party" can be the validator itself, if the field is not optional). The corresponding private key will be used to sign consensus messages on the validator's behalf, as well as sign for the validator in any other instances required by the protocol proper (i.e., besides user-initiated interactions with the auction and related system contracts). The link between the validator and the operator, as well as the history of validator's operator public keys, is to be tracked by the auction contract. Tracking of the pairings between a validator and its operators is intended to enable slashing, distribution of rewards and collection of transaction fees.
+
+For safety, consensus messages from a validator signed with validator's own private key ought to be ignored in an era where this validator has a designated operator.
+
+### Cases
+
+Genesis validators - operator specified at platform launch
+Entrant (no history as bidder or validator) - optionally supplies operator public key in `add_bid`
+Existing bid - changes bid data (by using `add_bid`, similar to adjusting bid token amount) to include an operator public key; platform will expect this operator to sign messages on validator's behalf in a future era where this bid wins
+
+### Typical timeline
+
+Bid entered by a new entrant B, no operator provided
+... (some blocks are added to the linear chain)
+Auction for era N + `auction_delay` takes place, B wins
+...
+Era N+auction_delay starts, B is a validator, signing own consensus messages with own private key
+During era N + `auction_delay`, B calls `add_bid` to supply an operator public key (and possibly add/reduce number of tokens in the bid)
+Auction for era N + `auction_delay` + `auction_delay` takes place, B wins again
```

Comment:
I think it's era N + `auction_delay` + `auction_delay` + 1. (Or "+ 2", if the first `add_bid` call is made in era N. Unless I'm making another off-by-one error here. :sweat_smile:)